multigen/pipes.py

```diff
@@ -188,6 +195,53 @@ def gen(self, inputs):
         return image
 
 
+class MaskedIm2ImPipe(Im2ImPipe):
+    _class = MaskedStableDiffusionImg2ImgPipeline
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._mask = None
+        self._image_painted = None
+        self._original_image = None
+        self._mask_blur = None
+
+    def setup(self, original_image=None, image_painted=None, mask=None, blur=4, **kwargs):
+        self._original_image = original_image
+        self._image_painted = image_painted
+        # there are two options:
+        # 1. mask is provided
+        # 2. mask is computed from difference between original_image and image_painted
+        if image_painted is not None:
+            neq = numpy.any(numpy.array(self._original_image) != numpy.array(self._image_painted), axis=-1)
+            mask = neq.astype(numpy.uint8) * 255
+        else:
+            assert mask is not None
+        self._mask = mask
+        self._image_painted = image_painted
+        input_image = image_painted if image_painted is not None else original_image
+        input_image = numpy.array(input_image)
```

Comment:
`numpy` is already imported as `np`. No need to import `numpy` once again and use it here.